### PR TITLE
doc: Explain Bitcoin Core instead of Bitcoin in README.md

### PR DESCRIPTION
**Pull request by maflcko**
Currently the README doesn't explain what Bitcoin Core is. Fix that.

Further reading / Inspired by:

* https://github.com/bitcoin/bitcoin/pull/25012 
* https://github.com/bitcoin-core/bitcoincore.org/pull/783
* https://github.com/bitcoin-core/bitcoincore.org/pull/784

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Bike shedding and NACKs welcome. 

The goal is, once and if this is merged, to propagate the description to the package metadata:

* https://github.com/bitcoin-core/packaging/blob/96794f2447e3357b847d879b53ead044d0e029b1/snap/snapcraft.yaml#L5
* https://github.com/flathub/org.bitcoincore.bitcoin-qt/blob/acad29d13868e98de48c2e4317294e5aa46d5a24/org.bitcoincore.bitcoin-qt.metainfo.xml#L21

**Comment by kristapsk** [comment]
Why not keep also "What is Bitcoin" part above "What is Bitcoin Core"?

**Comment by laanwj** [comment]
ACK fa0a6ed14d6a486b79a9fe7ede7ffb69e6527720

Motivation the same as I explained in #25012:

> "What is bitcoin" is very much up for discussion. There's no need to try to resolve, nor explain that here, definitely not anymore. Replace it by a short, nonambigious blurb what this project is.

**Comment by MarcoFalke** [comment]
> Why not keep also "What is Bitcoin" part above "What is Bitcoin Core"?

At this point, I don't think it is not up to this project to prescribe what Bitcoin is. If you mean a description, I am wondering what the goal of it would be, considering that (likely) anyone finding this project would already know about Bitcoin. See also https://github.com/bitcoin/bitcoin/pull/25012#issue-1218432612

**Comment by mzumsande** [comment]
Concept ACK 
Though I think that it would be good to at least mention in the title or description that this replaces the old description of Bitcoin (e.g. "explain Bitcoin Core instead of Bitcoin ...").

**Comment by MarcoFalke** [comment]
Thx, changed title

**Comment by cjfranca** [comment]
I understand Bitcoin is a coin and  bitcoin core is off the technologie.

File: README.md [hunk 1]
```diff
@@ -6,18 +6,14 @@ https://bitcoincore.org
...
+---------------------
 
-Bitcoin is an experimental digital currency that enables instant payments to
-anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
-with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
-software which enables the use of this currency.
+Bitcoin Core connects to the Bitcoin P2P network to download and fully validate
+blocks and transactions. It also includes a wallet and graphical user
+interface, that can be optionally built.
```
Comment (jonatack) [inline README.md:14]: suggestion: "a wallet and a graphical user interface (GUI), both of which can be optionally built."

**Review by brunoerg — COMMENTED** [review]
Strong Concept ACK

File: README.md [hunk 2]
```diff
@@ -6,18 +6,14 @@ https://bitcoincore.org
...
-----------------
+What is Bitcoin Core?
+---------------------
 
-Bitcoin is an experimental digital currency that enables instant payments to
-anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
-with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
-software which enables the use of this currency.
+Bitcoin Core connects to the Bitcoin P2P network to download and fully validate
```
[the remark this answers is not in the document]
Comment (MarcoFalke) [reply README.md]: Thanks, taken

File: README.md (hunk 1 above)
Comment (MarcoFalke) [reply README.md]: Thanks, switched to "which"

**Comment by laanwj** [comment]
re-ACK faeb5b59a098578b3e8c552d35b5ba02b12af14d

File: README.md [hunk 3]
```diff
@@ -6,18 +6,14 @@ https://bitcoincore.org
...
-----------------
+What is Bitcoin Core?
+---------------------
 
-Bitcoin is an experimental digital currency that enables instant payments to
-anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
-with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
-software which enables the use of this currency.
+Bitcoin Core connects to the Bitcoin peer-to-peer network to download and fully
```
Comment (prusnak) [inline README.md:12]: Nitpick: Maybe it's obvious, but shouldn't the question "What is X?" answered by a sentence that starts with "X is ..." before answering what X does?

```suggestion
Bitcoin Core is a software that connects to the Bitcoin peer-to-peer network to download and fully
```

Comment (laanwj) [reply README.md:12]: Sure, though given the context, I mean it's github, there's only software on here.

Comment (MarcoFalke) [reply README.md:12]: It is explained in the previous sentence: "the Bitcoin Core software", so I am going to reject the change for now.

**Review by brunoerg — APPROVED** [review]
ACK faeb5b59a098578b3e8c552d35b5ba02b12af14d

**Review by ghost — APPROVED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/25090/commits/faeb5b59a098578b3e8c552d35b5ba02b12af14d

**Comment by 1440000bytes** [comment]
> I understand Bitcoin is a coin and bitcoin core is off the technologie.

@cjfranca

_Since Bitcoin is both a currency and a protocol, capitalization can be confusing. Accepted practice is to use **Bitcoin (singular with an upper case letter B) to label the protocol**, software, and community, and bitcoins (with a lower case b) to label units of the currency._

https://en.bitcoin.it/wiki/Help:Introduction#Capitalization_.2F_Nomenclature

**Review by w0xlt — APPROVED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/25090/commits/faeb5b59a098578b3e8c552d35b5ba02b12af14d

**Comment by MarcoFalke** [comment]
Follow-ups in:

* https://github.com/bitcoin-core/packaging/pull/157
* https://github.com/flathub/org.bitcoincore.bitcoin-qt/pull/20

**Pull request merged**